test/0.8.25/vaults/contracts/Protocol__Deployment.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../../../contracts/0.8.9/EIP712StETH.sol";
+import "forge-std/Test.sol";
+
+import {CommonBase} from "forge-std/Base.sol";
+import {LidoLocator} from "../../../../contracts/0.8.9/LidoLocator.sol";
+import {StdCheats} from "forge-std/StdCheats.sol";
+import {StdUtils} from "forge-std/StdUtils.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {console2} from "forge-std/console2.sol";
+
+interface ILido {
+    function getTotalShares() external view returns (uint256);
+    function mintExternalShares(address _recipient, uint256 _amountOfShares) external;
+    function burnExternalShares(uint256 _amountOfShares) external;
+    function setMaxExternalRatioBP(uint256 _maxExternalRatioBP) external;
+    function initialize(address _lidoLocator, address _eip712StETH) external payable;
+    function resumeStaking() external;
+    function resume() external;
+    function setStakingLimit(uint256 _maxStakeLimit, uint256 _stakeLimitIncreasePerBlock) external;
+}
+
+interface IKernel {
+    function acl() external view returns (IACL);
+    function newAppInstance(bytes32 _appId, address _appBase, bytes calldata _initializePayload, bool _setDefault) external;
+}
+
+interface IACL {
+    function initialize(address _permissionsCreator) external;
+    function createPermission(address _entity, address _app, bytes32 _role, address _manager) external;
+    function hasPermission(address _who, address _where, bytes32 _what) external view returns (bool);
+}
+
+interface IDaoFactory {
+    function newDAO(address _root) external returns (IKernel);
+}
+
+contract Protocol__Deployment is Test {
+    ILido public lidoContract;
+
+    address private rootAccount;
+    address private userAccount;
+
+    address public kernelBase;
+    address public aclBase;
+    address public evmScriptRegistryFactory;
+    address public daoFactoryAdr;
+
+    address public accounting = makeAddr("dummy-locator:accounting");
+
+    function prepareLidoContract(uint256 _startBalance, address _rootAccount, address _userAccount) public {
+        rootAccount = _rootAccount;
+        userAccount = _userAccount;
+
+        vm.startPrank(rootAccount);
```

Comment:
Maybe separate those "pranks" to a separate methods for better readability? like setting up Kernet, setup dao, setup lido, etc...